WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -7157,12 +7157,6 @@
 				COPY_PHASE_STRIP = NO;
 				DEFINES_MODULE = YES;
 				ENABLE_BITCODE = YES;
-				FRAMEWORK_SEARCH_PATHS = (
```

Comment:
I don't think this change belongs in this PR as it's not related to the drag and drop... could you revert it here please? Is this to fix the 'incompatible objective-c category definitions' warning?